src/components/expanded-state/UniqueTokenExpandedState.tsx

```diff
@@ -230,7 +230,7 @@ const UniqueTokenExpandedState = ({ asset: passedAsset, external }: UniqueTokenE
   const { colors, isDarkMode } = useTheme();
   const { isReadOnlyWallet } = useWallets();
   const collectible = useCollectible(passedAsset?.uniqueId);
-  const asset = external ? passedAsset : collectible;
+  const asset = external ? passedAsset : (collectible as UniqueAsset);
```

Comment:
any chance that collectible is `undefined` here?

Reply:
I mostly just don't like casting types as it leads to unnecessary side effects. If we can't guarantee that the collectible is a type by type inference, it's usually a sign of bad type management.